<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorForm</class>
 <widget class="QWidget" name="SimulatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="UiPathLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="UiPathSelect">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rotate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="RotateSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>359</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="ViewScaleEnable">
           <property name="text">
            <string>FitInView</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="SimulatorView" name="UiSimulatorView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="BtnStart">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/build.png</normaloff>:/Menu/rc/Menu/build.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnPause">
           <property name="text">
            <string>Pause</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/pause.png</normaloff>:/Menu/rc/Menu/pause.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnStop">
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/stop.png</normaloff>:/Menu/rc/Menu/stop.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="TabAutoRunSet">
          <attribute name="title">
           <string>AutoRun</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Floor</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Arrow</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Function</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimulatorView</class>
   <extends>QGraphicsView</extends>
   <header location="global">simulatorform.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
